osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -255,5 +285,8 @@ def execute_task(extraction_order_id, supported_export_formats, execution_config
                     subprocess.check_call("docker-compose stop --timeout 0".split(' '))
                     subprocess.check_call("docker-compose rm -v -f".split(' '))
                 except:
+                    logger.exception("couldn't clean up docker containers in directory {directory}".format(
+                        directory=tmp_dir
+                    ))
                     pass
```

Comment:
Pass is in this case indeed redundant, since pass is a noop and only needed for the compiler not to say anything if a statement is required.